### PR TITLE
Finalizing Setup rework

### PR DESCRIPTION
**Pull request by lilyuwuu**
This PR reworks Finalizing Setup to (hopefully) make it easier to follow. The following changes have been made:

- The explanation of "valid arm9 payloads" run as `boot.firm` has been removed, as well as references to how SysNAND CFW bypasses the need for EmuNAND.
- The Move EmuNAND infobox has been moved to right before What You Need, and more explicitly says that it doesn't apply to most people.;
- The 2.1.0 CTRTransfer warning has been removed (I don't think an N3DS will even boot on 2.1 with boot9strap, and anyone in that situation would probably ask us for help anyway)
- "What You Need" has been split into three sections, depending on what the user is downloading (CIA files, 3DSX files, and "other")
    - Files are now in alphabetical order, both in What You Need and in Prep Work
- DSP1 is switched from 3DSX to CIA
- Prep Work has been condensed to include all file types in one step
- ctr-no-timeoffset and DSP1 have been combined into one step
- Explicit "Success" message at the very end
- Replace whatever was in "Information and Notes" before with an infobox containing keycombos, a link to the eiphax tips page, and a link to GodMode9 usage

Should be ready to merge, but any suggestions/tweaks/reversions are welcome.

### CODE REVIEW

File: _pages/en_US/finalizing-setup.txt [hunk 1]
```diff
@@ -57,12 +61,6 @@ During this process, we also setup programs such as the following:
 
 #### Section II - Updating the System
 
-If, while following a previous version of this guide, you CTRTransfered your *New 3DS* to 2.1.0, you should now [restore your NAND backup](godmode9-usage#restoring-a-nand-backup) before doing this section.
-{: .notice--danger}
```
Comment (lifehackerhansol) [inline _pages/en_US/finalizing-setup.txt:61]: Is this okay to remove?

**Comment by lifehackerhansol** [comment]
Looks okay other than that one burning question above.

File: _pages/en_US/finalizing-setup.txt (hunk 1 above)
Comment (lilyuwuu) [reply _pages/en_US/finalizing-setup.txt:61]: Resolved by re-word: "If your **New 3DS** was on firmware 2.1.0 before following this guide, you should [restore your NAND backup](godmode9-usage#restoring-a-nand-backup) before continuing."

**Review by lifehackerhansol — APPROVED** [review]

**Review by radostin04 — COMMENTED** [review]
I'm not sure how good the "CIA Files" and "3dsx" files headers are over the old method - I think it might get confusing for some helpees.

Maybe we might want to mention the filetype to the right of the link, even for those that only have one file?

**Comment by lilyuwuu** [comment]
I feel like that would be repetitive. People *should* be able to understand "**CIA Files** (get the `.cia` file)" ?

**Comment by radostin04** [comment]
I mean, remove the "CIA files", and keep just the (get the `.cia` file)

**Comment by lilyuwuu** [comment]
resolved

**Pull request merged**